code/game/machinery/computer/arcade/arcade_game.dm

```diff
@@ -0,0 +1,359 @@
+				feedback_inc("arcade_win_normal")
+				var/prizeselect = pickweight(prizes)
+				new prizeselect(holder.loc)
+
+				if(istype(prizeselect, /obj/item/toy/gun)) //Ammo comes with the gun
+					new /obj/item/toy/ammo/gun(holder.loc)
+
+				else if(istype(prizeselect, /obj/item/clothing/suit/syndicatefake)) //Helmet is part of the suit
+					new	/obj/item/clothing/head/syndicatefake(holder.loc)
+
+			else //admins can varedit arcades to have special prizes via contents, but it removes the prize rather than spawn a new one
+				feedback_inc("arcade_win_normal")
+				var/atom/movable/prize = pick(holder.contents)
+				prize.forceMove(holder.loc)
+
+	else if (emagged && (turtle >= 4))
+		var/boomamt = rand(5,10)
+		temp = "[enemy_name] throws a bomb, exploding you for [boomamt] damage!"
+		player_hp -= boomamt
+
+	else if ((enemy_mp <= 5) && (prob(70)))
+		var/stealamt = rand(2,3)
+		temp = "[enemy_name] steals [stealamt] of your power!"
+		player_mp -= stealamt
+		holder.updateUsrDialog()
+
+		if (player_mp <= 0)
+			gameover = 1
+			sleep(10)
+			temp = "You have been drained! GAME OVER"
+			if(emagged)
+				feedback_inc("arcade_loss_mana_emagged")
+				usr.gib()
+			else
+				feedback_inc("arcade_loss_mana_normal")
+
+	else if ((enemy_hp <= 10) && (enemy_mp > 4))
+		temp = "[enemy_name] heals for 4 health!"
+		enemy_hp += 4
+		enemy_mp -= 4
+
+	else
+		var/attackamt = rand(3,6)
+		temp = "[enemy_name] attacks for [attackamt] damage!"
+		player_hp -= attackamt
+
+	if ((player_mp <= 0) || (player_hp <= 0))
+		gameover = 1
+		temp = "You have been crushed! GAME OVER"
+		if(emagged)
+			feedback_inc("arcade_loss_hp_emagged")
+			usr.gib()
+		else
+			feedback_inc("arcade_loss_hp_normal")
+
+	blocked = 0
+
+/datum/arcade_game/space_villain/proc/action_charge()
+	blocked = 1
+	var/chargeamt = rand(4,7)
+	temp = "You regain [chargeamt] points"
+	player_mp += chargeamt
+	if(turtle > 0)
+		turtle--
+
+	holder.updateUsrDialog()
+	sleep(10)
+	arcade_action()
+
+/datum/arcade_game/space_villain/proc/action_heal()
+	blocked = 1
+	var/pointamt = rand(1,3)
+	var/healamt = rand(6,8)
+	temp = "You use [pointamt] magic to heal for [healamt] damage!"
+	holder.updateUsrDialog()
+	turtle++
+
+	sleep(10)
+	player_mp -= pointamt
+	player_hp += healamt
+	blocked = 1
+	holder.updateUsrDialog()
+	arcade_action()
+
+/datum/arcade_game/space_villain/proc/action_attack()
+	blocked = 1
+	var/attackamt = rand(2,6)
+	temp = "You attack for [attackamt] damage!"
+	holder.updateUsrDialog()
+	if(turtle > 0)
+		turtle--
+
+	sleep(10)
+	enemy_hp -= attackamt
+	arcade_action()
+
+/datum/arcade_game/space_villain/is_cheater(mob/user)
+	if(emagged && !gameover)
+		if(holder.stat & (NOPOWER|BROKEN))
+			return 0
+		else if(user in cheaters)
+			to_chat(usr, "<span class='danger'>[enemy_name] throws a bomb at you for trying to cheat him again.</span>")
+			explosion(get_turf(holder.loc),-1,0,2)//IED sized explosion
+			user.gib()
+			cheaters = null
+			qdel(src)
+			return 1
+		else
```

Comment:
I don't expect a proc called is_cheater to add the user to the list of cheaters but I'm also not sure this is worth changing

Reply:
It did this in the previous space villain itteration. It's an odd thing, and I'll probably change it when I add the second game.